sqs_workers/async_task.py

```diff
@@ -49,7 +52,8 @@ def delay(self, *args, **kwargs):
         _delay_seconds = kwargs.pop("_delay_seconds", None)
         _deduplication_id = kwargs.pop("_deduplication_id", None)
         _group_id = kwargs.pop("_group_id", None)
-        kwargs = bind_arguments(self.processor, args, kwargs)
+        if not self.queue.batching_policy.batching_enabled:
+            kwargs = bind_arguments(self.processor, args, kwargs)
```

Comment:
Same here. If we ignore `*args` when batching is enabled, please make it super explicit.

Reply:
Done as outlined in the earlier comment